test/browser_tests/conf.js

```diff
@@ -1,32 +1,118 @@
 'use strict';
 
-var env = require( './environment.js' );
-
-exports.config = {
-  multiCapabilities: [
-    {
-      browserName: 'chrome',
-      specs:       [ 'spec_suites/shared/*.js' ]
-    },
-    {
-      browserName: 'firefox',
-      specs:       [ 'spec_suites/shared/*.js' ]
-    },
-    // Large Screen only tests
-    {
-      browserName:   'chrome',
-      specs:         [ 'spec_suites/large_screen/*.js' ],
-      chromeOptions: {
-        args: [ '--lang=en',
-                '--window-size=1200,900' ]
+var settings = require( './settings.js' );
+var defaultSuites = require( './default-suites.js' );
+
+function _paramIsSet( param ) {
+  return typeof param !== 'undefined';
+}
+
+/*
+* Params that need to be passed to protractor config.
+*/
+function _retrieveProtractorParams( params ) { // eslint-disable-line complexity, no-inline-comments, max-len
+  var parsedParams = {};
+
+  var checkParam = params.specs;
```

Comment:
1. Yeah, the lookup isn't costing you anything here, it's not like this is run in production or millions of times and it's a lot clearer to read this way.

2. I'm going to have to respond to this one outside of this outdated diff because it's hiding everything now.